packages/dashboard/src/components/MultiCount/MultiCount.tsx

```diff
@@ -55,24 +64,87 @@ export class MultiCount extends React.Component<IMultiCountProps, {}> {
           annotationTextFontSize: annotationTextFontSize,
           annotationTextColor: annotationTextColor,
           bodyTextColor: bodyTextColor,
-          bodyTextFontSize: bodyTextFontSize
+          bodyTextFontSize: bodyTextFontSize,
+          hoveredText: this.state.hoveredText,
+          currentText: row.data + row.bodyText + row.annotaionText,
+          href: row.href
         })
       );
+      const expandingCardProps: IExpandingCardProps = {
+        onRenderCompactCard: this._onRenderCompactCard,
+        renderData: row,
+        mode: ExpandingCardMode.compact,
+        styles: {
+          compactCard: {
+            width: 'auto',
+            height: 'auto'
+          }
+        }
+      };
+      const hoverKey = row.data + row.bodyText + row.annotaionText;
       formattedRows.push(
-        <div key={index} className={classNames.root}>
-          <div className={classNames.bodyText}>
-            <span className={classNames.data}>{formattedData + units[indexForUnits]}</span>
-            <span>{row.bodyText}</span>
+        <HoverCard
+          key={index}
+          expandingCardProps={expandingCardProps}
+          instantOpenOnClick={true}
+          onCardHide={this._hoverStateUpdate.bind(this, hoverKey, false)}
+          onCardVisible={this._hoverStateUpdate.bind(this, hoverKey, true)}
+        >
+          <div key={index} className={classNames.root} onClick={this._redirectToUrl.bind(this, row.href)}>
+            <div className={classNames.bodyText}>
+              <span className={classNames.data}>{formattedData + units[indexForUnits]}</span>
+              <span>{row.bodyText}</span>
+            </div>
+            <div className={classNames.annotationText}>
+              <span className={classNames.icon}>
+                <Icon iconName={changeIconIndicator} />
+              </span>
+              <span className={classNames.annotationText}>{row.annotaionText}</span>
+            </div>
           </div>
-          <div className={classNames.annotationText}>
-            <span className={classNames.icon}>
-              <Icon iconName={changeIconIndicator} />
-            </span>
-            <span className={classNames.annotationText}>{row.annotaionText}</span>
-          </div>
-        </div>
+        </HoverCard>
       );
     });
     return formattedRows;
   }
+
+  private _hoverStateUpdate = (hoverKey: string, hoverState: boolean): void => {
+    if (hoverState) {
+      this.setState({
+        hoveredText: hoverKey
+      });
+    } else {
+      this.setState({
+        hoveredText: ''
+      });
+    }
+  };
+
+  private _redirectToUrl(href: string | undefined): void {
+    href ? (window.location.href = href) : '';
+  }
+
+  private _onRenderCompactCard = (data: IMultiCountRow): JSX.Element => {
+    const changeIconIndicator =
+      AnnotationType[data.type] === 'neutral' ? '' : AnnotationType[data.type] === 'positive' ? 'FlickDown' : 'FlickUp';
```

Comment:
>positive [](start = 84, length = 8)

are you sure these will be lower case? Do you want to compare this with tolower?